### DISCUSSION TITLE
`--rdp-range_middle-foreground-color` var name

### DISCUSSION THREAD

**Discussion by lfernandes-cci** [gpbl/react-day-picker#2607] [Ideas]
`--rdp-range_middle-foreground-color` in the documentation should be `--rdp-range_middle-color` to change the color of range text for day picker

**Comment by gpbl** [comment]
This is a great idea! I have no idea why I used the “foreground” color term… 😅

**Reply by lfernandes-cci** [reply]
@gpbl , Just the documentation says "foreground" but "--rdp-range_middle-color" works while actually applying the styles in code. It took some trial and error to figure it out. The documentation will need this change.
